byzcoin/proto.go

```diff
@@ -57,7 +57,7 @@ type CreateGenesisBlock struct {
 	Version Version
 	// Roster defines which nodes participate in the skipchain.
 	Roster onet.Roster
-	// GenesisDarc defines who is allowed to write to this skipchain.
+	// AdminDarc defines who is allowed to write to this skipchain.
```

Comment:
What?

Reply:
When I wanted to connect to the DEDIS-ByzCoin, you created a DARC for me, which is now _my_ admin darc. So the darc stored in the config is not necessarily the genesis-darc, but only any admin darc that is allowed to do things in this ByzCoin instance.

Reply:
Right, but the field is still called GenesisDarc, so the documentation string needs to start with, at least, a capital G...

Reply:
Thanks - now I got it. Yes, it's IntelliJ over-zealously replacing `GenesisDarc` with `AdminDarc`, and me not reading the code on Friday afternoon...